interfaces/builtin/u2f_devices.go

```diff
@@ -62,6 +62,11 @@ var u2fDevices = []u2fDevice{
 		VendorIDPattern:  "096e|2ccf",
 		ProductIDPattern: "0880",
 	},
+	{
+		Name:             "HyperSecu HyperFIDO Pro",
+		VendorIDPattern:  "2ccf",
+		ProductIDPattern: "0854",
```

Comment:
https://devicehunt.com/view/type/usb/vendor/2CCF#search-results-table I only see 0880 in the device table, maybe we should add both paterns?

Reply:
Note that 0880 is actually already in our table, in the one right above. Maybe we should join that one with this one, though it doesn't really make any actual difference

Reply:
Yeah, I guess we could extend the existing HyperSecu entry and have 0880|0854 in the product ID pattern. The `Name` field is only informational anyway.

Reply:
Actually I forgot about the Name section, in this case I think it makes sense to leave `0880` in the one above, since that corresponds to the non-Pro version. I think this is PR as-is is actually fine, nvm me